Misc/NEWS.d/next/Library/2024-12-06-21-03-11.gh-issue-127688.NJqtc-.rst

```diff
@@ -0,0 +1 @@
+Add the ``SCHED_DEADLINE`` and ``SCHED_NORMAL`` constants to the :mod:`os` module.
```

Comment:
```suggestion
Add the :data:`~os.SCHED_DEADLINE` and :data:`~os.SCHED_NORMAL`` constants to the :mod:`os` module.
```

As usual, since it's a new feature, let's add a What's New entry.